### PR TITLE
Brian/save a workout

### PR DESCRIPTION
**Pull request by briankimlim**
general UI changes/additions for Saving a workout from the WorkoutActivity screen, no DB interaction yet.

### CODE REVIEW

**Comment by jiaweini** [comment]
I tested your build. The "Choose this gym" button isn't showing on this page. Everything else seems fine.

image

File: app/src/main/res/layout/activity_gym_preset.xml [hunk 1]
```diff
@@ -1,50 +1,44 @@
...
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/availableEquip" />
+        android:gravity="center_horizontal"
+        android:layout_marginTop="8dp" />
 
-    <TextView
-        android:id="@+id/availableEquip"
-        android:layout_width="412dp"
+    <Button
+        android:id="@+id/gym_confirm_button"
```
Comment (jiaweini) [inline app/src/main/res/layout/activity_gym_preset.xml:41]: Might want to add this back 
`android:layout_marginBottom="48dp"`

**Review by jiaweini — DISMISSED** [review]

File: app/src/main/res/layout/activity_gym_preset.xml (hunk 1 above)
Comment (briankimlim) [reply app/src/main/res/layout/activity_gym_preset.xml:41]: That was due to absolute positioning margins but I replaced them with relative positioning. Hopefully it works on all devices now.

**Review by bendwilletts — APPROVED** [review]
This looks good to me, merge when ready

**Pull request merged**